controllers/suite_test.go

```diff
@@ -162,9 +162,9 @@ func newLoadTest() *grpcv1.LoadTest {
 				Name:     optional.StringPtr("driver"),
 				Language: "cxx",
 				Pool:     optional.StringPtr("drivers"),
-				Run: grpcv1.Run{
-					Image: optional.StringPtr("gcr.io/grpc-test-example/driver:v1"),
-				},
+				Run: []corev1.Container{{
+					Image: *optional.StringPtr("gcr.io/grpc-test-example/driver:v1"),
```

Comment:
Do we need *optional.StringPtr? This is just a string. Same for the other places where this appears.